backend/src/apiserver/client_manager/client_manager.go

```diff
@@ -71,6 +74,18 @@ const (
 	clientBurst = "ClientBurst"
 )
 
+var scheme *runtime.Scheme
+
+func init() {
```

Comment:
there is already a clientmanager init step, is there benefit to doing this out of that context? if so maybe a comment here might help future devs

Reply:
It's a common pattern in Kubernetes controllers so that the code panics when the package is imported if there is a coding error. Guaranteed to encounter it if it's at all imported in a test.